pilot/pkg/networking/util/util.go

```diff
@@ -247,6 +247,12 @@ func SortVirtualHosts(hosts []route.VirtualHost) {
 	})
 }
 
+// IsProxyVersionGE13 checks whether the given Proxy version is greater than or equals 1.3.
+func IsProxyVersionGE13(node *model.Proxy) bool {
+	ver, _ := node.GetProxyVersion()
```

Comment:
Thanks for adding this.

Reply:
Related to my other comment, do we actually also need to update https://github.com/istio/istio/blob/a7e57b226268c7e0b2365c646683cc95d7babbbb/pilot/docker/Dockerfile.proxyv2#L47 to make this really work?

If so I would prefer to do it in this PR to make sure the e2e tests is running with the latest code change, and catch fails right in this PR if there is any.